test/acceptance/integration/ISSUE-322 Service support for registries (NGSI9)/issue-322.feature

```diff
@@ -29,7 +29,7 @@ Feature: Service support for registries (NGSI9)
 
   Scenario: Subscribe different entities with the same provider and subservices but different service
     # First registration
-    Given a new NGSI version "9" petition with the service "issue_322" and the subservice "/subservice"
+    Given a new NGSI "9" api petition with the service "issue_322" and the subservice "/subservice"
```

Comment:
petition -> request ?

Probably this change should be done in all the tests (using grep to locate all ocurrences)